plaso/parsers/syslog_plugins/cron.py

```diff
@@ -32,7 +32,8 @@ class CronSyslogPlugin(interface.SyslogPlugin):
   """A syslog plugin for parsing cron messages."""
 
   NAME = 'cron'
-  DESCRIPTION = 'Parser for syslog cron messages.'
+  DATA_FORMAT = 'Cron syslog file'
```

Comment:
Isn't this rather for "cron-relevant lines in a syslog file" ?

Reply:
ack, changed file into line